packages/js/e2e-core-tests/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 
 ## Added
 
+- This package is now transpiled with Babel, which allows the usage of modern, yet compatible JS code.
```

Comment:
It's okay to leave this entry for this PR. Package change log entries are for changes that affect package users vs a log of PR changes. The package being transpiled does not affect anyone outside this repository so there is no need for this entry.

Reply:
> As mentioned in one of my other comments

Sorry, it seems this comment wasn't saved.